lab_1/main.py

```diff
@@ -14,6 +28,26 @@ def tokenize(text: str) -> list:
     """
     pass
 
+    if not isinstance(text, str):
+        return []
+    text = text.lower()
+    text = text.replace('\n', ' ')
+    letters = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u',
+               'v', 'w', 'x', 'y', 'z']
+    new_text = ''
+    for symbol in text:
+        if symbol in letters or symbol == ' ':
+            new_text += symbol
+    tokens = new_text.split()
+    return tokens
+
+
+tokens = tokenize(text)
+
+
+stop_words = read_from_file('stop_words.txt')
```

Comment:
There should be no function calling\using in `main.py` -> use functions on the`start.py`.